### ISSUE TITLE
Documentation: `analyzer` argument in Atlas search index is required

### ISSUE THREAD

**Issue by darknessest** [mongodb/terraform-provider-mongodbatlas#1132]
### Terraform CLI and Terraform MongoDB Atlas Provider Version

```hcl
Terraform v1.3.9
MongoDB Atlas Provider v.1.8.2
```

### Terraform Configuration File 

```hcl
resource "mongodbatlas_search_index" "my-atlas-index" {
  name            = "my-atlas-index"
  project_id      = "[redacted]"
  cluster_name    = "mycluster"
  database        = "mydb"
  collection_name = "mycoll"

  analyzers = jsonencode(
    [
      {
        "name" : "standardExample",
        "tokenFilters" : [
          {
            "type" : "lowercase"
          }
        ],
        "tokenizer" : {
          "type" : "standard"
        }
      }
    ]
  )

  mappings_dynamic = false
  mappings_fields = jsonencode(
  {
    "created_at" : {
      "indexDoubles" : false,
      "representation" : "int64",
      "type" : "number"
    },
    "text" : {
      "analyzer" : "standardExample",
      "store" : false,
      "type" : "string"
    }
  }
)
}
```

### Steps to Reproduce

1. `terraform plan -input=false`

### Expected Behavior

Either the documentation should state that `analyzer` is a required argument, or it should actually default to "lucene.standard".

### Actual Behavior

While running a `terraform plan` it actually says that the argument is required.

### Debug Output 

Here's the `terraform plan` output

```
Terraform v1.3.9
on linux_amd64
Initializing plugins and modules...
╷
│ Error: Missing required argument
│
│   on mongo-index.tf line 1, in resource "mongodbatlas_search_index" "my-atlas-index":
│    1: resource "mongodbatlas_search_index" "my-atlas-index" {
│
│ The argument "analyzer" is required, but no definition was found.
╵
Operation failed: failed running terraform plan (exit 1)
```

### Additional Context

Everything is executed in a Github action with [hashicorp/setup-terraform@v2](https://github.com/hashicorp/setup-terraform)

**Comment by martinstibbe** [comment]
@darknessest Confirmed TF side does not default it to a value and docs do not state optional just mentions default value
Will correct in internal engineering ticket [INTMDB-780](https://jira.mongodb.org/browse/INTMDB-780)

**Comment by Zuhairahmed** [comment]
hi @darknessest  we have released fix to this issue as part of v1.10.0 release. can you test and let us know if this issue has been resolved?

**Comment by darknessest** [comment]
@Zuhairahmed, seems like everything is good now, thank you!
